### PR TITLE
fixing UI documentation

### PR DESCRIPTION
**Pull request by pasiasty**
Updating documentation by the information of setting PointerScope.Canvas in case of using both HTML GUI and Excalibur. Fixed typo in code snippet.

### CODE REVIEW

**Comment by eonarheim** [comment]
@excaliburjs/core-contributors Should we make `PointerScope.Canvas` pointer scope the default? This seems like the most useful/common case?

**Comment by eonarheim** [comment]
@kamranayub Seems to be a credential issue with the CI, was this related to PRs?

File: docs/14-ui.mdx [hunk 1]
```diff
@@ -56,6 +56,8 @@ A [screen element](#screen-elements) is useful because it's an actor, it is rend
 
 The trick is _positioning the game UI above Excalibur_ and reacting to Excalibur events appropriately. With some tweaks to the way you embed Excalibur, this can be accomplished without too much code.
 
+Do not forget to set [PointerScope.Canvas](https://excaliburjs.com/docs/api/edge/enums/_input_pointer_.pointerscope.html#canvas), otherwise Excalibur will capture all mouse input and HTML GUI won't work properly.
```
Comment (kamranayub) [inline docs/14-ui.mdx:59]: This can actually be shortened to `[[PointerScope.Canvas]]`

Comment (pasiasty) [reply docs/14-ui.mdx]: Sure thing! Done.

**Review by eonarheim — APPROVED** [review]
@pasiasty thanks! Looks good 👍

**Pull request merged**